indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -207,6 +207,42 @@
                         </li>
                     </ul>
                 </div>
+                <div class="group">
+                    <a class="i-button arrow icon-copy js-requires-selected-row single-row hide-if-locked" data-toggle="dropdown">
+                        {% trans %}Clone{% endtrans %}
+                    </a>
+                    <ul class="i-dropdown">
+                        <li>
+                            <a class="js-dialog-action"
+                               title="{% trans %}Add new abstract{% endtrans %}"
+                               data-title="{% trans %}Add new abstract{% endtrans %}"
```

Comment:
Maybe these titles should mention cloning?

Reply:
Sure, I didn't initially because it could be confused with a straight clone, instead of a new form.
However, maybe something like "Clone into an abstract" or just "Clone abstract"?

Reply:
Now that I'm revising this, I don't think the titles make sense. Because after all, the UI and actions are the exact same as adding a new abstract, it's just the information that gets pre-filled, as if someone was writing it manually (+ new i18n strings). Do you think we should really change them?

<img width="836" alt="imagem" src="https://user-images.githubusercontent.com/12183954/150413320-2978a8f5-9297-423c-89a0-bc08760c46de.png">


<img width="836" alt="imagem" src="https://user-images.githubusercontent.com/12183954/150413228-af32c34c-ac0e-4413-93df-f6f662d4f7c3.png">

Reply:
hm... no strong opinion tbh - so i'll leave it up to you (or maybe get some more opinions in our MM channel)